docs/serving/model-serving-get-started.ipynb

```diff
@@ -850,6 +850,117 @@
     "- Steps B and C are filter steps that complement each other. For example B passes events where key < X, and C passes events where key >= X. The resulting DF contains the exact event ingested, since each event was handled once on one of the branches.\n",
     "- Steps B and C modify the content of the event in different ways. B adds a column col1 with value X, and C adds a column col2 with value X. The resulting DF contains both col1 and col2. Each key is represented twice: once with col1 == X, col2 == null and once with col1 == null, col2 == X."
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Multi-process (concurrent) steps\n",
+    "Multi-process steps are typically used for serving of deep-learning models, where preparation steps and inference can be CPU heavy.\n",
```

Comment:
Could also be GPU heavy, or involving I/O (then a `concurrency_mechanism` other than `"multiprocessing"` should be used).

Reply:
I also think the concurrency mechanisms need to be explained, as per [the docstring](https://github.com/mlrun/storey/blob/v1.7.23/storey/flow.py#L939).